Documentation/Communicational Patterns/2. Tooling/2.5. Environment.md

```diff
@@ -6,15 +6,17 @@ The `@Environment` property wrapper provides different functionality depending o
 
 ## ObservableObject
 
-If the `wrappedValue` conforms to `ObservableObject`, the `@Environment` behaves exactly the same as an [`@ObservedObject`](./2.4.%20ObservedObject.md) that was passed into the `Handler` from outside.
-
-In any other case `@Environment` has no influence on the `Handler`'s lifetime or control flow.
+If the `wrappedValue` conforms to `ObservableObject`, the `@Environment` can be used to change or read its properties. The `ObservableObject` will be injected to the `Handler` by using the corresponding keypath. This action may trigger other `Handler`s or `Job`s when changing properties annotated with `@Published`.
```

Comment:
I would add an explicit statement that an `ObservedObject` injected using `@Environment` doesn't trigger the `handle` method.